pkg/controller/collector/collector_controller.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 	"time"
 
-	loggingv1 "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
```

Comment:
there's an established practice here of using this naming convention to make it clear you're referencing the external types (and of course if we ever introduce a v2 api), i don't see a good reason to change that practice.

Reply:
Those aren't external types - that's why I changed it. Its our own logging API. I think this was just a typo, every other source file uses "logging"